cost/azure/rightsize_managed_disks/README.md

```diff
@@ -0,0 +1,61 @@
+
+- It is important that you keep constant LUN numbers for your data disks because when policy retrieves metrics for them, it will only look for the latest LUN number assigned to the data disk, if this one is changed multiple times before running the policy, only the data points using the current LUN the disk has will be retrieved for analysis.
+
+### Policy Saving Details
+
+The policy includes the estimated monthly savings. The estimated monthly savings are recognized if the resource is terminated.
+
+- The `Estimated Monthly Savings` is calculated by obtaining the price of the disk per month from the Azure Pricing API.
+- The incident message detail includes the sum of each resource `Estimated Monthly Savings` as `Potential Monthly Savings`.
+
+## Input parameters
+
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created.
+- *Azure Endpoint* - The endpoint to send Azure API requests to. Recommended to leave this at default unless using this policy with Azure China.
+- *Minimum Savings Threshold* - Minimum potential savings required to generate a recommendation.
+- *Exclusion Tags (Key:Value)* - Cloud native tags to ignore resources that you don't want to produce recommendations for. Use Key:Value format for specific tag key/value pairs, and Key:\* format to match any resource with a particular key, regardless of value. Examples: env:production, DO_NOT_DELETE:\*
+- *Allow/Deny Subscriptions* - Determines whether the Allow/Deny Subscriptions List parameter functions as an allow list (only providing results for the listed subscriptions) or a deny list (providing results for all subscriptions except for the listed subscriptions).
+- *Allow/Deny Subscriptions List* - A list of allowed or denied Subscription IDs/names. If empty, no filtering will occur and recommendations will be produced for all subscriptions.
+- *Allow/Deny Regions* - Whether to treat Allow/Deny Regions List parameter as allow or deny list. Has no effect if Allow/Deny Regions List is left empty.
+- *Allow/Deny Regions List* - Filter results by region, either only allowing this list or denying it depending on how the above parameter is set. Leave blank to consider all the regions.
+- *Minimum used disk space percentage to consider a managed disk for downsizing* - The minimum used disk space percentage threshold at which to consider a disk to be 'oversized' and therefore be flagged for downsizing.
```

Comment:
Name of parameter should be shorter and capitalised. The description is for the additional details.  This feedback applies to the other parameters and the policy itself as well.

Example of how it should look:

- *Minimum Used Disk Space (%)* - The minimum used disk space percentage threshold at which to consider a disk to be 'oversized' and therefore be flagged for downsizing.